packages/core/src/toast/Toast.css

```diff
@@ -28,8 +28,12 @@
 }
 
 /* Styles applied to icon */
-.saltToast-icon.saltIcon {
+.saltToast-iconContainer > .saltIcon {
   margin: calc(var(--salt-spacing-75) + 2px) 0 var(--salt-spacing-75) 0;
+  color: var(--toast-borderColor);
+}
+.salt-density-high.saltToast-iconContainer > .saltIcon {
```

Comment:
`.salt-density-high.saltToast-iconContainer` will never apply on a single `div`?

I don't think we should do density specific calculations in component. Does token themselves are wrong?